Formula/aptos.rb

```diff
@@ -0,0 +1,43 @@
+class Aptos < Formula
+  desc "Universal and fair access to decentralized assets for billions of people"
```

Comment:
maybe this would be more accurate?
```suggestion
  desc "Layer 1 blockchain built to support the widespread use of blockchain"
```

Reply:
Adapted your suggestion and modified it